e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc.yml

```diff
@@ -46,6 +46,44 @@ spec:
               value: "30100"
             - name: CLIENT_PORT_MAX
               value: "30105"
+            - name: PMM_AGENT_SERVER_ADDRESS
+              value: monitoring-service
+            - name: PMM_AGENT_LISTEN_PORT
+              value: "7777"
+            - name: PMM_AGENT_PORTS_MIN
+              value: "30100"
+            - name: PMM_AGENT_PORTS_MAX
+              value: "30105"
+            - name: PMM_AGENT_SETUP_NODE_ADDRESS
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: status.podIP
+            - name: PMM_AGENT_SETUP_CONTAINER_ID
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.name
```

Comment:
It should be Docker container ID (`/docker/123…`) for compatibility with cAdvisor. See https://confluence.percona.com/x/ppGUAg

Reply:
We use POD_NAME for PMM_AGENT_SETUP_NODE_NAME and as I can see containerID contains not unique value (in our case), every restart we will have new containerID .

Reply:
> as I can see containerID contains not unique value (in our case), every restart we will have new containerID

So that's actually unique and what is needed

Reply:
ok, but we can have the situation that e.g. `pxc` container is restarted but `pmm-client ` not. In this case we will have outdated containerID.

Reply:
How can you restart one container in a pod without restarting the whole pod?

Reply:
You can not (no, you have one way `kill mysqld`) but container can be re-started by itself `exit status > 0` or liveness probe can restart it.